policy/phosimCmptSetting.yaml

```diff
@@ -13,6 +13,10 @@ intraDirName: intra
 # camera piston.
 extraDirName: extra
 
+# Wavefront Sensor directory name. This is needed if the defocal image is
+# provided by the wavefront sensors.
+wfsDirName: wfs
```

Comment:
If this is the directory name, where is the path (or root) in the file system? Or this is compared with the root of this repository?

Reply:
This is where PhoSim works and saves the images within each `iterX/img` directory.

Reply:
Please help to add the description to the comment. Thanks!